akka-stream/src/main/scala/akka/stream/actor/ActorPublisher.scala

```diff
@@ -102,8 +102,8 @@ object ActorPublisherMessage {
  * requested elements.
  *
  * Create a [[org.reactivestreams.Publisher]] backed by this actor with Scala API [[ActorPublisher#apply]],
- * or Java API [[UntypedActorPublisher#create]] or Java API compatible with lambda expressions
- * [[AbstractActorPublisher#create]].
+ * or Java API [[ClassicActorPublisher#create]] or Java API compatible with lambda expressions
+ * [[ClassicActorPublisher#create]].
```

Comment:
I can't find `ClassicActorPublisher` in this PR but that one I think we can also not rename

Reply:
Just making sure you were paying attention. I meant to check that, thanks for catching it.